features/bevragen/persoon/verblijfplaats/onbekend/fields.feature

```diff
@@ -0,0 +1,120 @@
+#language: nl
+
+Functionaliteit: verblijfplaats onbekend velden vragen met fields
+
+  Achtergrond:
+    Gegeven de persoon met burgerservicenummer '000000152' heeft de volgende 'verblijfplaats' gegevens
+    | naam                               | waarde   |
+    | land adres buitenland (13.10)      | 0000     |
+    | datum aanvang adreshouding (10.30) | 20150808 |
+    | datum ingang geldigheid (85.10)    | 20220222 |
+    | functie adres (10.10)              | W        |
+    En de persoon met burgerservicenummer '000000153' heeft de volgende 'verblijfplaats' gegevens
+    | naam                                   | waarde   |
+    | land adres buitenland (13.10)          | 0000     |
+    | datum aanvang adres buitenland (13.20) | 20160808 |
+    | datum ingang geldigheid (85.10)        | 20200222 |
+
+  Abstract Scenario: 'datum aanvang adreshouding (10.30)' wordt gevraagd met field pad '<fields>'
+    Als personen wordt gezocht met de volgende parameters
+    | naam                | waarde                          |
+    | type                | RaadpleegMetBurgerservicenummer |
+    | burgerservicenummer | 000000152                       |
+    | fields              | <fields>                        |
+    Dan heeft de response een persoon met de volgende 'verblijfplaats' gegevens
+    | naam                 | waarde                 |
+    | type                 | VerblijfplaatsOnbekend |
+    | datumVan.type        | Datum                  |
```

Comment:
Hier doet zich denk ik een vreemde situatie voor. Je levert 'Type' op basis van de waarde '0000' in land adres buitenland (13.10). Tegelijkertijd lever je de 'datumVan' op basis van datum aanvang adreshouding (10.30) wat gerelateerd is aan verblijfplaatsBinnenland. Is dat niet vreemd?

Je zou zeggen lever een van de volgende 2 opties:
* 'datumVan' en 'Type' met de waarde 'Adres'
* 'Type' met de waarde 'verblijfplaatsOnbekend'